lib/exercises.rb

```diff
@@ -2,11 +2,38 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: n*m*1 (n = number of strings, m = number of characters in each string, 1 for the lookups on the hash)
+# Space Complexity: depends on the number of strings with distinct sets of letters, worst case n for a hash key for each string
 
 def grouped_anagrams(strings)
```

Comment:
👍 Nice work!  The space complexity is O(n) since you're making a hash of all the words.